packages/lib/shared/components/btns/button-group/LabelWithIcon.tsx

```diff
@@ -0,0 +1,21 @@
+import { HStack, Text } from '@chakra-ui/react'
+import { PropsWithChildren } from 'react'
+import { SignIcon } from '../../icons/SignIcon'
+import { GasIcon } from '../../icons/GasIcon'
+
+type Props = {
```

Comment:
We could pass the icon as a ReactNode to make it more flexible but keeping it simple for now.